manifests/ingress.yaml

```diff
@@ -4,15 +4,18 @@ kind: Ingress
 metadata:
   name: bennu-app
   namespace: bennuhp
+  annotations:
+    cert-manager.io/issuer: "letsencrypt"
 spec:
+  tls:
+    - hosts:
+        - bennu-official.page
+      # cert-manager automatically generate secret
+      secretName: tls-cert
   ingressClassName: "nginx"
-  defaultBackend:
-    service:
-      name: nginx
-      port:
-        number: 80
   rules:
-  - http:
+  - host: bennu-official.page
```

Comment:
note: we have to add custom domain for applying ingress rules. All IP adderess in A record will point same static IP, which we had created from Google Cloud with `gcloud compute addresses create ...` command.